install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -34,27 +40,71 @@ storage provider.
 ====
 
 [[ceph-provisioning]]
-
 == Provisioning
+
 Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision Ceph volumes in OpenShift, the
-following are required:
+a volume in OpenShift. To provision Ceph volumes in OpenShift, the following
+information is required:
 
-- Existing Ceph volumes, to be defined in the persistent volume object
-- The `*PersistentVolume*` API
+- The ceph key (in base64) to be used as an OpenShift secret object.
+- The ceph image name.
+- The file system type on top of the block storage (e.g., ext4).
 
-You must also ensure *ceph-common* package is installed on all OpenShift nodes in the cluster:
+You must also ensure that the *ceph-common* package is installed on *_all_*
+schedulable OpenShift nodes:
 
 ----
 # yum install ceph-common
 ----
 
+[[creating-ceph-secret]]
+=== Creating the Ceph Secret
+
+In a secret definition, the storage administrator must define the admin authorization key,
```

Comment:
# openshift version
openshift v3.1.1.6
kubernetes v1.1.0-origin-1107-g4c8e6f4
etcd 2.1.2

-- creating pod with non-base64 secret results in the pod failing with this mount error:
2s		6	{kubelet rhel7.2-dev}			FailedMount	Unable to mount volumes for pod "ceph-pod1_default": fork/exec /usr/bin/rbd: invalid argument

Changing nothing else except for base64 encoding the secret works.
@pmorie Perhaps this enhancement was added to a newer verison of openshift?